blog/on_retros/README.md

```diff
@@ -0,0 +1,272 @@
+
+If your organization change management process is ok with it, I'd suggest allowing for paired PRs to merge immediately after tests pass, though of course you should still circulate the PR to allow other's the opportunity to stay informed or make suggestions, even if they are after merging a PR. You can always revert it or do a follow-up if someone brings up something smart two hours later!
+
+### Bottlenecks in UAT/Testing/QA
+
+Once code is flowing, the next place tickets tend to bottleneck is in the UAT and QA/Testing stage.
+
+On the ecommerce team at Peloton, we require that applicable tickets (read: anything that changes production behavior visible to users/customers in production, i.e. not refactoring) requires production signoff. All tickets that resulted in code changing requires "testing" signoff.
+
+Most teams struggle with this because PMs are really busy and it's hard for them to be on top of everything all the time. Depending on the ticket, sometimes we allow engineering managers to do product sign off.
+
+Some things we have done in retros to help with this are
+1) Write simple testing instructions in the ticket when it is ready for PM review, explaining the workflow and how to test, so the PM can signoff async. If the ticket is simple enough, maybe you just want to link example orders/records/whatever that they can verify after, or you can take screenshots.
+2) On another team, we introduced the concept of a "mini retro" at 4:50 each day when we could first quickly bring up any new issues/blockers we wanted to write down before standup the next day, but also we could use the last 5-10 minutes of the day to demo any of the work we got done. That way the whole team could actually see what was being accomplished and product could sign off on the spot, with only a 15-minute daily commitment.
+
+For QA/Testing, we were running into issues where the "tester," being another handoff, need to be retold all the decisions and context for the change, shown the workflow, shown the unit tests, shown the implementation, etc. This was a huge bottleneck.
+
+Some things that worked well for us were
+1) Decided that instead of requiring signoff from "the SDET," instead any engineer who didn't make a commit could signoff, but the SDET would work with the engineers to teach them what the standards were for testing.
```

Comment:
+10000000

SDET role is not a phase gate. They are an engineer on the team.

Want to turn it up to 11?

Kill all the roles. Everyone is a developer - I MEAN EVERYONE